module/VuFind/src/VuFind/Controller/EPFController.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+/**
+ * EPF Controller
+ *
+ * PHP version 8
+ *
+ * Copyright (C) Villanova University 2010.
```

Comment:
You might want to check dates in your new files and use 2023 where appropriate.

Reply:
Crap, I did that in a few places but forgot (many) others.  Fixed.